pages/linux/mocp.md

```diff
@@ -0,0 +1,40 @@
+# mocp
+
+> Music on Console (MOC) audio player.
+> More information: <http://moc.daper.net/>.
+
+- Launch MOC terminal UI:
+
+`mocp`
+
+- Launch MOC terminal UI in specific directory:
+
+`mocp {{path/to/directory}}`
+
+- Launch MOC in background:
+
+`mocp &`
```

Comment:
This isn't really `mocp`-specific, so we usually don't list it in the tldr pages. Also, removing it would solve the problem, that a page shouldn't have more than 8 examples.